test/DynamoCoreWpfTests/PackageManager/PackageManagerUITests.cs

```diff
@@ -2272,6 +2272,46 @@ public void AssertPreviewPackageRetainFolderStructureEqualsPublishLocalPackageRe
             Directory.Delete(publishPath, true);
         }
 
+        [Test]
+        public void AssertPreviewPackageRetainFolderStructureEqualsPublishLocalPackageResultsForNestedFolders()
+        {
+            var packageName = "NestedPackage";
+            var pathManager = this.ViewModel.Model.PathManager as PathManager;
+            var publishPath = Path.Combine(pathManager.DefaultPackagesDirectory, packageName);
+
+            string nodePath = Path.Combine(TestDirectory, "pkgs", packageName);
+            var allFiles = Directory.GetFiles(nodePath, "*", SearchOption.AllDirectories).ToList();
```

Comment:
if you intended to add a test package with binaries, you did not commit them (you'll need to force add them) - but if you can avoid that I would because it will likely cause other test failures.

Reply:
Ok, will do